client/utils/charge/index.ts

```diff
@@ -120,3 +121,15 @@ export const getChargeAmounts = ( charge: Charge ): ChargeAmounts => {
 
 	return balance;
 };
+
+export const getChargeChannel = ( type: string ): string => {
+	if (
+		type === 'card_present' ||
+		type === 'interac_present' ||
+		type === 'in_person'
```

Comment:
Not sure where `in_person` might come from, can you elaborate a little bit @elazzabi ?

Reply:
Yes! I added a small comment on the line above https://github.com/Automattic/woocommerce-payments/pull/4392#discussion_r924218971. I can change course (or add a code comment) if it can clear things up

Reply:
Thanks! Can we place the comment right in the codebase?

Reply:
Sounds good! I added a comment to the method https://github.com/Automattic/woocommerce-payments/pull/4392/commits/54d99d10acac7529dbb6374de186599658bbe71b